docs/AcceptableUnicode.md

```diff
@@ -0,0 +1,92 @@
+# Acceptable Unicode characters:
+Using Unicode seems to be irresistible. However, we must ensure
+avoiding problematic Unicode usage. Here is a list of acceptable
+and forbidden Unicode characters and accents. Please note that
+we forbid the use of accents, because this can lead to visually
+ambiguous characters (see below).
+
+## Acceptable lower-case Greek letters:
+α # (alpha)
+β # (beta)
+δ # (delta)
+ϵ # (epsilon)
+γ # (gamma)
+κ # (kappa)
+λ # (lambda)
+μ # (mu)
+η # (eta)
+ω # (omega)
+π # (pi)
+ρ # (rho)
+σ # (sigma)
+θ # (theta)
+χ # (chi)
+ξ # (xi)
+ζ # (zeta)
+ϕ # (psi)
+φ # (varphi)
+
+## Acceptable upper-case Greek letters:
+Δ # (Delta)
+∑ # (Sigma)
+Γ # (Gamma)
+Ω # (Omega)
+Φ # (Phi)
+Ψ # (Psi)
+
+## Acceptable mathematical symbols:
+∫ # (int)
+∬ # (iint)
+∭ # (iiint)
+∈ # (in)
```

Comment:
Are we avoiding:
```
∉ # \notin
≠ # \ne
≉ # \napprox
```
The later two are hard to distinguish I think

Reply:
Also both `∈ # (in)` and `ϵ # (epsilon)` can be hard to distinguish in some fonts.

Reply:
Yeah, I suggest that we avoid those negated symbols. I have an umbrella statement at the end of the doc, I'll move it to the beginning. And good point about `∈ # (in)` and `ϵ # (epsilon)`. I'm torn. My vote would be to ban `in` to be more consistent with allowing greek letters.